GitHubRepositories.ps1

```diff
@@ -1607,3 +1607,530 @@ function Move-GitHubRepositoryOwnership
 
     return Invoke-GHRestMethod @params
 }
+
+Function Get-GitHubRepositoryVulnerabilityAlert {
```

Comment:
Normal pattern is to have the noun part of the function name consistent across related functions, so this should then be `Test-GithubRepositoryVulnerabilityAlert` to match with `Enable-GithubRepositoryVulnerabilityAlert` and `Disable-GithubRepositoryVulnerabilityAlert`.

Reply:
Yeah, I was thinking about that.  On the one hand it makes sense to follow the pattern, on the other hand, a function named `Test-GitHubRepositoryVulnerabilityAlert` seemed like it might try testing a specific alert.  It wasn't clear if what it did would be clear.  Following naming convention with clear CBH might be sufficient to handle that concern.